src/rabbit_peer_discovery_cleanup.erl

```diff
@@ -0,0 +1,306 @@
+%% The contents of this file are subject to the Mozilla Public License
+%% Version 1.1 (the "License"); you may not use this file except in
+%% compliance with the License. You may obtain a copy of the License
+%% at http://www.mozilla.org/MPL/
+%%
+%% Software distributed under the License is distributed on an "AS IS"
+%% basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See
+%% the License for the specific language governing rights and
+%% limitations under the License.
+%%
+%% The Original Code is RabbitMQ.
+%%
+%% The Initial Developer of the Original Code is AWeber Communications.
+%% Copyright (c) 2015-2016 AWeber Communications
+%% Copyright (c) 2016-2017 Pivotal Software, Inc. All rights reserved.
+%%
+-module(rabbit_peer_discovery_cleanup).
+
+-behaviour(gen_server).
+
+-include_lib("rabbitmq_peer_discovery_common/include/rabbit_peer_discovery.hrl").
+
+-export([start_link/0,
+         check_cluster/0]).
+
+%% gen_server callbacks
+-export([init/1,
+         handle_call/3,
+         handle_cast/2,
+         handle_info/2,
+         terminate/2,
+         code_change/3]).
+
+-ifdef(TEST).
+-compile(export_all).
+-endif.
+
+-define(CONFIG_MODULE, rabbit_peer_discovery_config).
+-define(CONFIG_KEY, node_cleanup).
+
+-define(CONFIG_MAPPING,
+        #{
+          cleanup_interval   => #peer_discovery_config_entry_meta{
+                                   type          = integer,
+                                   env_variable  = "CLEANUP_INTERVAL",
+                                   default_value = 60
+                                  },
+          cleanup_only_log_warning  => #peer_discovery_config_entry_meta{
```

Comment:
Ok, sorry I got it now.
If I want to disable it, just don't configure [`node_cleanup`](https://github.com/rabbitmq/rabbitmq-peer-discovery-common/pull/1/files#diff-8a84a237f7532b279a26d94f1b3cdce9R95)
Pardon.